lib/publiccloud/instance.pm

```diff
@@ -848,5 +848,25 @@ sub do_systemd_analyze_time {
     return @ret;
 }
 
+sub retry_on_script_output {
+    my ($self, %args) = @_;
+    my $retries = 10;
+    my $delay_command = 20;
+    my $sys_response = "";
+
+    while ($retries--) {
+        $sys_response = $self->ssh_script_output(
+            cmd => 'sudo systemctl is-system-running',
```

Comment:
Uneeded: it is already done in the original code loop from L425-849, long studied, tested, checked, reviewed and correctly running until now.